python/tests/test_account.py

```diff
@@ -1622,6 +1632,7 @@ def test_ephemeral_timer(self, acfactory, lp):
 
         lp.sec("ac2: set ephemeral timer to 0")
         chat2.set_ephemeral_timer(0)
+        ac2._evtracker.get_matching("DC_EVENT_CHAT_EPHEMERAL_TIMER_MODIFIED")
```

Comment:
This is for another PR probably?

Reply:
it's just adding more precision to the test, don't want to do a new PR just for this line.